### PR TITLE
Update contracts for new assets

### PR DESCRIPTION
**Pull request by wagyu298**
It contains the following changes:

- add skills and remove accessories from contracts
- change token name and symbols
- modify contractURI to return JSON via data URI

### CODE REVIEW

**Comment by wagyu298** [comment]
Tests still fails until #5 merged.

**Comment by wagyu298** [comment]
Sorry, I missunderstood the domain URL.
I consider to rebase to clear wrong URLs before merging. Because don't store invalid URLs in the develop branch.

File: packages/nouns-contracts/contracts/NounsArt.sol [hunk 1]
```diff
@@ -271,60 +278,60 @@ contract NounsArt is INounsArt {
...
-     * @notice Add a batch of accessory images from an existing storage contract.
+     * @notice Add a batch of head images from an existing storage contract.
      * @param pointer the address of a contract where the image batch was stored using SSTORE2. The data
      * format is expected to be like {encodedCompressed}: bytes created by taking a string array of
      * RLE-encoded images, abi encoding it as a bytes array, and finally compressing it using deflate.
      * @param decompressedLength the size in bytes the images bytes were prior to compression; required input for Inflate.
-     * @param imageCount the number of images in this batch; used when searching for images among batches.
-     * @dev This function can only be called by the descriptor.
+     * @param imageCount the number of images in this batch; used when searching for images among batches
+     * @dev This function can only be called by the descriptor..
```
Comment (ms76) [inline packages/nouns-contracts/contracts/NounsArt.sol]: I think it's original, but is this a typo? `..`

**Review by ms76 — COMMENTED** [review]
@wagyu298 I've reviewed and confirmed the behavior.

I commented on one point.
And is it correct that the `*ContractURIHash` below should be `*ContractURI`?

https://github.com/CNNouns/cnnouns-monorepo/blob/39c56886603acae3374046227507edbd6137faf7/packages/nouns-contracts/abi/contracts/NounsToken.sol/NounsToken.json#L948-L960

Could you please check?

**Comment by wagyu298** [comment]
@ms76 Thank you for your coding reviews. I fixed a typo. 

>  And is it correct that the *ContractURIHash below should be *ContractURI?

I update ABIs on pull request #9 .

**Review by ms76 — APPROVED** [review]
@wagyu298 I've reviewed your revision. LGTM!

> I update ABIs on pull request https://github.com/CNNouns/cnnouns-monorepo/pull/9 .

Yes, it was. I have confirmed it in #9.

**Pull request merged**